README.md

```diff
@@ -14,6 +14,8 @@ Frontend for the new supporter platform: [https//support.theguardian.com/](https
 8. [A/B Test framework](/docs/development.md#8-ab-test-framework)
 9. [Test environments](/docs/development.md#9-test-environments)
 10. [CSS guidelines](/docs/development.md#10-css-guidelines)
+11. [Payment Diagrams](/docs/development.md#11-payment-diagrams)
```

Comment:
What is this? Was it missing before?

Reply:
Yeap, it is a useful diagram that @jranks123 created for PayPal. It would be nice to create it for other payment methods.